modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/websocket/WebSocketAPITestCase.java

```diff
@@ -356,6 +357,14 @@ public void configure(WebSocketServletFactory factory) {
      * @param accessToken API accessToken
      */
     private void testThrottling(String accessToken) throws Exception {
+
+        final int UNIT_TIME = 5; //5 minutes (The unit time is defined as 5 minutes in policy.json)
+        /* Prevent API requests getting dispersed into two time units */
+        while (LocalDateTime.now().getMinute() % UNIT_TIME >= (UNIT_TIME - 1)) {
```

Comment:
Isn't this condition reduces to == , rather than >=

Reply:
The functionality would be the same if we use '=='
But '==' doesn't cover that the values less than that should return false. In the sense of logic, it is more appropriate to use >= AFAIU , hence code readability would be better.